x509/cert_pool.go

```diff
@@ -128,3 +128,7 @@ func (s *CertPool) Subjects() (res [][]byte) {
 	}
 	return
 }
+
+func (s *CertPool) Contains(c *Certificate) bool {
```

Comment:
This is pretty poor form to inject into an independent library. If this isn't yet implemented, this needs to panic, or we shouldn't add it. But as it stands, it's pretty dangerous and unneeded to add this here.